### ISSUE TITLE
does not connect someone else's address book

### ISSUE THREAD

**Issue by PbIC9TuHa** [jobisoft/DAV-4-TbSync#97]
## Your environment

TbSync version: 0.7.16
DAV-4-TbSync version: 0.10
Thunderbird version: 52.2.0 **(32-bit)**

## Expected behavior
public address book connects the link 
http://myserver-abook/caldav.php/**public**/addresses/
...

## Actual behavior
I try to connect public address book, but only personal is connected , that which is attached to my login
http://myserver-abook/caldav.php/**LOGIN**/addresses/
But I'm clearly pointing out that I need the address book from http://myserver-abook/caldav.php /**PUBLIC**/ addresses

but he insists I connect only the personal address book.
tried different versions of the plugin , the result is the same. Unfortunately, it is not possible to change the TB version.
I will try on a separate stand the latest versions of TB and plug - in-I will update about the result
...

## Steps to reproduce
...

To help resolving your issue, enable debug logging (TbSync Account Manager -> Help) and send me the debug.log via e-mail (use the title of your issue as subject of the email).

**Comment by jobisoft** [comment]
Is any other client automatically finding these public address books, when you set up your account?

If your server is sending these public address books when asked for resources accociated with your account, than TbSync needs to learn how to get them. I will not implement an option to manually add addressbooks / URLs.

I am generally interested in discovering public resources, if I someone points me to an RFC or some other hints.

**Comment by PbIC9TuHa** [comment]
Good afternoon. I've only tried evolution and Thunderbird.When creating an account, we specify which address book we want to connect. For example, the user has a personal Address book and a public one. For example, this address personal AddBook http://myserver-abook/caldav.php/LOGIN_USER/addresses/ book connects perfectly everywhere, both evolution and Thunderbird. If we want to connect Public AddBook then link http://myserver-abook/caldav ahhh!php/PUBLIC/addresses/ Evolution connects fine , but Thunderbird with the plugins TbSync+DAV-4-TbSync why it still connects http://myserver-abook/caldav.php/LOGIN_USER/addresses/ in the prescribed address public links. Thus, the user gets 2 Addbooks exactly the same.I tried to connect only one public book , but to no avail.I'm not asking you to redo the plugin, just maybe tell me what I'm doing wrong ? The impression that the plugin does not read links to AddBook. maybe  need some parameters in the INI files to add ?
And pardon my English - Google seeks to as can :)

**Comment by jobisoft** [comment]
Could you try to setup an account using the first option (Discover Service) and just enter "myserver-abook" as server (whatever that stands for) and do not add any thing behind? No /caldav.php etc.

If you tell me, what "myserver-abook" actually stands for, I could test your server to find out, what you have to use as server address. But in most cases, using just the host name works. That is the core feature of TbSync.

**Comment by PbIC9TuHa** [comment]
Good afternoon. There is a server in our local network which I for an example called "myserver-abook" . There's no outside access. It has a server installed https://www.davical.org/ . I tried as you said Discovery Service. Unfortunately no changes , but as I began to understand the problem is not in the plugin - in the settings of my server "myserver-boom ". The question is closed. Sorry for the time taken
